.vscodeignore

```diff
@@ -46,3 +45,4 @@ tsconfig*.json
 tsconfig*.tsbuildinfo
 webpack.config*.mjs
 yarn.lock
+.wdio-vscode-service
```

Comment:
This doesn't hurt anything, but this already isn't getting bundled today.

Reply:
> but this already isn't getting bundled today

I see that this folder is not git ignored, and I see it in the bundled zip
![Screenshot 2024-12-18 at 10 22 00](https://github.com/user-attachments/assets/8fdf3cc1-8eae-44c9-8567-890ad583e961)

> This doesn't hurt anything

Maybe it's truth in the environment that is isolated from tests running, but I see that it may affect the final size

![Screenshot 2024-12-18 at 10 23 12](https://github.com/user-attachments/assets/46fc517d-2774-45f1-a786-4210ef49ceb0)

Reply:
Not sure what is creating that folder -- we aren't using webdriver, is that just an old folder you have? When I said it won't be bundled, I meant via CI (your local folder can have all kinds of stuff that would get "accidentally" bundled)